--- libfprint-0.4.0/configure.ac.bero	2013-01-15 23:27:32.327835468 +0100
+++ libfprint-0.4.0/configure.ac	2013-01-15 23:27:38.380762364 +0100
@@ -2,7 +2,7 @@ AC_INIT([libfprint], [0.4.0])
 AM_INIT_AUTOMAKE([1.11 dist-bzip2 no-dist-gzip check-news])
 AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_SRCDIR([libfprint/core.c])
-AM_CONFIG_HEADER([config.h])
+AC_CONFIG_HEADERS([config.h])
 
 # Enable silent build when available (Automake 1.11)
 m4_ifdef([AM_SILENT_RULES],[AM_SILENT_RULES([yes])])
